Szwalnia/Element_szczegoly.cs

```diff
@@ -26,11 +26,10 @@ public Element_szczegoly(SzwalniaEntities db, int ID)
             {
                 txtOkres.Text = element.Okres_Przydatnosci_Miesiace.ToString();
             }    
-            txtIlosc.Text = element.Element_Ilosc_W_Paczce.ToString();
-            txtJednostka.Text = db.Elementy_Jednostki.Where(unit => unit.ID_jednostka == element.ID_Jednostka).First().Jednostka;
+            //txtJednostka.Text = db.Elementy_Jednostki.Where(unit => unit.ID_jednostka == element.ID_Jednostka).First().Jednostka;
```

Comment:
unikamy commitowania zakomentowanego kodu

Reply:
usunięto zakomentowany kod